lib/eventFactory.js

```diff
@@ -501,7 +510,8 @@ class EventFactory extends BaseFactory {
                             pipeline: p,
                             pipelineConfig: modelConfig,
                             changedFiles,
-                            webhooks
+                            webhooks,
+                            prInfo
```

Comment:
Need to change jsdoc to include `prInfo`

Reply:
yup, just noticed that